index.d.ts

```diff
@@ -108,7 +108,15 @@ declare module '@cloudflare/workers-types' {
     get(key: string, type?: 'stream'): Promise<ReadableStream>
     get(key: string, type?: 'text' | 'json' | 'arrayBuffer' | 'stream'): Promise<any>
 
-    put(key: string, value: string | ReadableStream | ArrayBuffer | FormData): Promise<void>
+    put(
+      key: string, 
+      value: string | ReadableStream | ArrayBuffer | FormData,
+      options?: { 
+        expiration?: string | number;
+        expirationTtl?: string | number;
+        [propName: string]: any;
```

Comment:
Why the index signature? This would allow a key like `expirationTTL`

Reply:
Problematic, true. To account for unknown options? I couldn't find comprehensive Worker KV API docs, so wasn't sure if there are other options.

I guess better to leave that out and add options as we're aware of them. Yes?